cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/Location.java

```diff
@@ -291,34 +314,34 @@ public boolean equals(Object o) {
         }
         Location location = (Location) o;
         return Objects.equals(this.name, location.name) &&
```

Comment:
do you think this auto-gen not producing equals ignore case is an issue?

Reply:
I can't think of a case where we would want that in this layer of the code.

Reply:
When RADAR storage is supported in the radar client, then it will matter. Storage/update of "RADAR TEST", "Radar Test", and "radar test" will all be for the same location and retrieval will only return one of them. By not introducing it now, you create an expectation of case sensitive ID support which is not the case.

If you retrieve "RADAR TEST" and get back "Radar Test" equality tests should pass whereas right now they will fail. Speaking of, do you actually test that?